lib/active_merchant/billing/gateways/stripe_payment_intents.rb

```diff
@@ -213,16 +213,22 @@ def purchase(money, payment_method, options = {})
       end
 
       def capture(money, intent_id, options = {})
-        post = {}
-        currency = options[:currency] || currency(money)
-        post[:amount_to_capture] = localized_amount(money, currency)
-        if options[:transfer_amount]
-          post[:transfer_data] = {}
-          post[:transfer_data][:amount] = options[:transfer_amount]
+        if intent_id.include?('pi_')
```

Comment:
It would be better just to do a `return Response.new(false, error_message) unless intent_id.include?('pi_')` here. Then the error message could be more generic like `"Only Authorizations performed via the Payment Intent API are  capturable”`